tests/Valit.Tests/Strategies/FailFast.cs

```diff
@@ -68,7 +68,7 @@ public void FailFast_Strategy_Checks_Until_First_Error_While_Error_Is_Not_First(
 
             result.Succeeded.ShouldBe(false);
             result.ErrorMessages.Count().ShouldBe(1);
-            result.ErrorMessages.ShouldContain(M2);
+            result.ErrorMessages.ShouldContain(M3);
```

Comment:
M2 == "Field is required". M3 == "This value should be at least N long." 

Same for any other changes similar to this in tests.